### PR TITLE
Create skole

### PR DESCRIPTION
**Pull request by nculumovic**
@skole.hr is available to primary and secondary school attendees in Croatia as well as educational staff in those schools

### CODE REVIEW

**Comment by zweizwei** [comment]
the portal doesn't seem to belong to any particular school. Please let me know if someone but students and staff can register the email on the website. Thanks!

**Comment by nculumovic** [comment]
That is correct, the portal does not belong to a particular school. The portal is maintained by CARNet, Croatian Academic and Research Network, under control by Ministry of Science Education and Sports. Info: http://www.carnet.hr/mission_vision
CARNet provides e-mail services only to those qualified for their electronic identity, as described here http://www.carnet.hr/en/webmail and here http://www.carnet.hr/en/electronic_identity . A person (pupil, student) is granted identity by it's home institution (school, university) based on their status (employed educational staff, active students and such). And in order for the home institution to become a member of CARNet it must be verified by the Minister of Science Education and Sports, as stated here: http://www.carnet.hr/en/membership.

tl:dr
Only active pupils students and educational staff are granted (they don't register) email on the domain, and they can use it only while enrolled in school, or working as part of the educational staff. Once that condition is no longer fulfilled their access is revoked. The whole process is controlled by a government funded agency, and Ministry of Science Education and Sports

tl:dr:tl:dr
None but students and staff can register

**Comment by zweizwei** [comment]
tx a lot for a thorough explanation - that's exactly what i needed. the file doesn't have .txt extension so i've committed the correct one separately - you'll be able to apply within half an hour. i'm closing this commit then.
